cms/djangoapps/contentstore/views/item.py

```diff
@@ -781,10 +781,17 @@ def create_xblock_info(xblock, data=None, metadata=None, include_ancestor_info=F
         visibility_state = None
     published = modulestore().has_published_version(xblock) if not is_library_block else None
 
-    #instead of adding a new feature directly into xblock-info, we should add them into override_type.
-    override_type = {}
+    # defining the default value 'True' for delete, drag and add new child actions in xblock_actions for each xblock.
+    xblock_actions = {'deletable': True, 'draggable': True, 'childAddable': True}
+    explanatory_message = None
     if getattr(xblock, "is_entrance_exam", None):
```

Comment:
@asadiqbal08 -- I've left this one as an exercise for you -- let me know if you need a hand.